packages/react-cosmos-component-playground/src/components/LoadersGrid/index.jsx

```diff
@@ -0,0 +1,9 @@
+import React, { Component } from 'react';
+
+class LoadersGrid extends Component {
```

Comment:
Thanks for bringing this, I did feel need to talk about it. After your proposal I see two different sets of responsibility:

LoadersGrid
---
To `LoadersGrid` you say: "here is a list of [component, fixture, label] triples" and it will:
- create many iframes
- take care of communication with them (send fixtures)
- layout those iframes and display labels
- provide an interface for changing from horizontal list to grid view

Would `LoadersManager` be a better name?

It would be an abstract tool for handling group views, situations where we render more than one component. It would be needed again on "project page" with all components rendered using their default fixture (also described in https://github.com/react-cosmos/react-cosmos/issues/314)

ComponentPage
---

I see a need for `ComponentPage` also. It would:
- prepare data for `LoadersGrid` [{component, fixture, description}, ...]
- handle information display for a component without fixtures
- host other component page on functionalities in the future